PlayerMethods.h

```diff
@@ -2209,7 +2209,7 @@ namespace LuaPlayer
         player->SetSummonPoint(summoner->GetMapId(), x, y, z);
 
         WorldPacket data(SMSG_SUMMON_REQUEST, 8 + 4 + 4);
-        data << uint64(summoner->GetGUIDLow());
+        data << summoner->GET_GUID();
```

Comment:
Guid is a different number from guid low. One of the codes likely has a bug.

Reply:
I know. Low part of guid is wrong here - raw value is needed (you can observe that by looking at the same packet in different cores). But for players, low part = raw value (highguid for players is 0) - no bug here.